chronos/apps/http/httpserver.nim

```diff
@@ -44,11 +44,11 @@ type
     Empty, Prepared, Sending, Finished, Failed, Cancelled, Dumb
 
   HttpProcessCallback* =
-    proc(req: RequestFence): Future[HttpResponseRef] {.gcsafe.}
+    proc(req: RequestFence): Future[HttpResponseRef] {.gcsafe, raises: [Defect].}
```

Comment:
This is not needed annotation because all the exceptions handled here https://github.com/status-im/nim-chronos/blob/master/chronos/apps/http/httpserver.nim#L589-L598

Reply:
fixed